tests/quotes/strings.js

```diff
@@ -31,6 +31,22 @@
 // Unnecessary escapes.
```

Comment:
This file needs to start with something other than a string - as written, these are all directives, not strings.

Also, you need to make sure this substitution *isn't* applied to directives; see https://github.com/prettier/prettier/issues/1555.